### PR TITLE
dev => master 2.2.0

### PR DESCRIPTION
**Pull request by edmundmiller**
Didn't get it out in time for the template update 😞

### CODE REVIEW

File: .github/workflows/download_pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install git+https://github.com/nf-core/tools.git@dev
+
+      - name: Get the repository name and current branch set as environment variable
+        run: |
+          echo "REPO_LOWERCASE=${GITHUB_REPOSITORY,,}" >> ${GITHUB_ENV}
+          echo "REPOTITLE_LOWERCASE=$(basename ${GITHUB_REPOSITORY,,})" >> ${GITHUB_ENV}
+          echo "REPO_BRANCH=${GITHUB_REF#refs/heads/}" >> ${GITHUB_ENV}
```
Comment (mashehu) [inline .github/workflows/download_pipeline.yml:45]: this is unfortunately broken in this version of the template
```suggestion
         
          echo "{% raw %}REPO_BRANCH=${{ github.event.inputs.testbranch || 'dev' }}" >> ${GITHUB_ENV}
```

File: .github/workflows/download_pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          --download-configuration
+
+      - name: Inspect download
+        run: tree ./${{ env.REPOTITLE_LOWERCASE }}
+
+      - name: Run the downloaded pipeline
+        env:
+          NXF_SINGULARITY_CACHEDIR: ./
+          NXF_SINGULARITY_HOME_MOUNT: true
+        run: nextflow run ./${{ env.REPOTITLE_LOWERCASE }}/$( sed 's/\W/_/g' <<< ${{ env.REPO_BRANCH }}) -stub -profile test,singularity --outdir ./results
```
Comment (mashehu) [inline .github/workflows/download_pipeline.yml:67]: Same here
```suggestion
        run: nextflow run ./${{ env.REPOTITLE_LOWERCASE }}/$( sed 's/\W/_/g' <<< ${{ env.REPO_BRANCH }}) -stub -profile test,singularity --outdir ./results{% endraw %}
```

File: README.md [hunk 3]
```diff
@@ -47,34 +48,47 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
+<!-- TODO nf-core: Describe the minimum required steps to execute the pipeline, e.g. how to prepare samplesheets.
+     Explain what rows and columns represent. For instance (please edit as appropriate):
+
+First, prepare a samplesheet with your input data that looks as follows:
 
-1. Install [`Nextflow`](https://www.nextflow.io/docs/latest/getstarted.html#installation) (`>=22.10.1`)
+`samplesheet.csv`:
 
-2. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) (you can follow [this tutorial](https://singularity-tutorial.github.io/01-installation/)), [`Podman`](https://podman.io/), [`Shifter`](https://nersc.gitlab.io/development/shifter/how-to-use/) or [`Charliecloud`](https://hpc.github.io/charliecloud/) for full pipeline reproducibility _(you can use [`Conda`](https://conda.io/miniconda.html) both to install Nextflow itself and also to manage software within pipelines. Please only use it within pipelines as a last resort; see [docs](https://nf-co.re/usage/configuration#basic-configuration-profiles))_.
+```csv
```
Comment (mashehu) [inline README.md:63]:
```suggestion
```csv title="samplesheet.csv
```

File: README.md [hunk 4]
```diff
@@ -47,34 +48,47 @@ On release, automated continuous integration tests run the pipeline on a full-si
...
+> [!NOTE]
+> If you are new to Nextflow and nf-core, please refer to [this page](https://nf-co.re/docs/usage/installation) on how to set-up Nextflow. Make sure to [test your setup](https://nf-co.re/docs/usage/introduction#how-to-run-a-pipeline) with `-profile test` before running the workflow on actual data.
+
+<!-- TODO nf-core: Describe the minimum required steps to execute the pipeline, e.g. how to prepare samplesheets.
+     Explain what rows and columns represent. For instance (please edit as appropriate):
+
+First, prepare a samplesheet with your input data that looks as follows:
 
-1. Install [`Nextflow`](https://www.nextflow.io/docs/latest/getstarted.html#installation) (`>=22.10.1`)
+`samplesheet.csv`:
```
Comment (mashehu) [inline README.md:61]:
```suggestion
```
just add it to the code block, see below

File: assets/nf-core-nascent_logo_light.png
Comment (mashehu) [inline assets/nf-core-nascent_logo_light.png:1]: something feels off here, can you try to recreate that logo

File: docs/output.md [hunk 5]
```diff
@@ -352,6 +370,7 @@ They've also created some bed files that might be useful for analysis.
   - `bwa/`: Directory containing bwa indices.
   - `bwa-mem2/`: Directory containing bwa-mem2 indices.
   - `dragmap/`: Directory containing DRAGMAP indices.
+  - `bowtie2/`: Directory containing DRAGMAP indices.
```
Comment (mashehu) [inline docs/output.md:373]:
```suggestion
  - `bowtie2/`: Directory containing bowtie2 indices.
```

File: main.nf [hunk 6]
```diff
@@ -18,21 +17,41 @@ nextflow.enable.dsl = 2
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
+// TODO nf-core: Remove this line if you don't need a FASTA file
+//   This is an example of how to use getGenomeAttribute() to fetch parameters
+//   from igenomes.config using `--genome`
```
Comment (mashehu) [inline main.nf:22]:
```suggestion
```

**Review by mashehu — APPROVED** [review]

**Review by alyssa-ab — APPROVED** [review]
LGTM! (other than maybe the failed linting tests)

**Pull request merged**